packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -528,7 +535,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
             ref={ this._splitButtonContainer }
             data-is-focusable={ true }
             onClick={ !disabled && !primaryDisabled ? this._onSplitButtonPrimaryClick : undefined }
-            tabIndex={ !disabled ? 0 : undefined }
+            tabIndex={ 0 }
```

Comment:
shouldn't this be conditional in the same way that data-is-focusable is conditional above? We can't expect all buttons to be in focus zones.

Reply:
tabIndex only takes type "number | undefined" so can't do a conditional boolean like with data-is-focusable

Reply:
I'm confused why this couldn't be set to 0 or undefined similar to data-is-focusable?